keyboards/inett_studio/sqx/hotswap/rules.mk

```diff
@@ -2,28 +2,24 @@
 MCU = atmega32u4
 
 # Bootloader selection
-#   Teensy       halfkay
-#   Pro Micro    caterina
-#   Atmel DFU    atmel-dfu
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
 BOOTLOADER = atmel-dfu
 
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = lite 	# Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
 MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = yes 		# Audio control and System control
+EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-NKRO_ENABLE = yes 			# USB Nkey Rollover
-UNICODE_ENABLE = no         # Unicode
-RGBLIGHT_ENABLE = yes        # Use RGB underglow light
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 RGB_MATRIX_ENABLE = yes     # Use RGB per key light
-LTO_ENABLE = yes			# Reducing firmware size
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+LTO_ENABLE = yes            # Reducing firmware size
```

Comment:
```suggestion
LTO_ENABLE = yes            # Reducing firmware size

LAYOUTS = 60_tsangan_hhkb
```

Reply:
```
$ make inett_studio/sqx/hotswap
QMK Firmware 0.9.52
Making inett_studio/sqx/hotswap with keymap bcat                                                       [OK]
Making inett_studio/sqx/hotswap with keymap default                                                    [OK]
Making inett_studio/sqx/hotswap with keymap default_60_tsangan_hhkb                                    [OK]
Making inett_studio/sqx/hotswap with keymap dohmain                                                    [OK]
Making inett_studio/sqx/hotswap with keymap via                                                        [OK]
Making inett_studio/sqx/hotswap with keymap yanfali                                                    [ERRORS]

 * The firmware is too large! 29548/28672 (876 bytes over)

make[1]: *** [tmk_core/rules.mk:411：check-size] 错误 1
Making inett_studio/sqx/hotswap with keymap yanfali_wkl                                                [ERRORS]
cc1.exe: error: users/yanfali_wkl: not a directory [-Werror]
cc1.exe: all warnings being treated as errors
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:386：.build/obj_inett_studio_sqx_hotswap_yanfali_wkl/keyboards/inett_studio/sqx/hotswap/hotswap.o] 错误 1
Make finished with errors
make: *** [Makefile:584：inett_studio/sqx/hotswap] 错误 1
```

Reply:
That's fine.  As long as the two default keymaps, and the via keymap are compiling fine, then there are no issues with the PR, even if travis CI fails. 

Eg, this is a natural issue with community layouts.